CHANGELOG.rst

```diff
@@ -14,6 +14,14 @@ Change Log
 Unreleased
 ----------
 
+[0.40.0] - 2017-08-08
+---------------------
+
+* Add Enterprise API Gateway for new Enterprise Catalogs and Programs endpoint.
+* Add /enterprise/api/v1/enterprise_catalogs/ endpoint.
+* Add /enterprise/api/v1/enterprise_catalogs/{uuid}/ endpoint.
```

Comment:
@UmanShahzad We still have underscores here. Should probably turn them into dashes after fbdf6222b8925de0ed2b4d2d302d014fb03fbdd4.